src/ceph.in

```diff
@@ -308,7 +308,8 @@ def parse_cmdargs(args=None, target='') -> Tuple[argparse.ArgumentParser,
     parser.add_argument('--cluster', help='cluster name')
 
     parser.add_argument('--admin-daemon', dest='admin_socket',
-                        help='submit admin-socket commands (\"help\" for help)')
+                        help='submit admin-socket commands (send "help\" for' \
```

Comment:
maybe without the 'send'? I would be confused by that.

Reply:
I pushed a change, PTAL